src/bitcoin-cli.cpp

```diff
@@ -33,6 +33,8 @@ const std::function<std::string(const char*)> G_TRANSLATION_FUN = nullptr;
 static const char DEFAULT_RPCCONNECT[] = "127.0.0.1";
 static const int DEFAULT_HTTP_CLIENT_TIMEOUT=900;
 static const bool DEFAULT_NAMED=false;
+static const int DEFAULT_GENERATED_NBLOCKS=1;
+static const int DEFAULT_GENERATED_MAXTRIES=100000;
```

Comment:
This is now unused, right?

Reply:
DEFAULT_GENERATED_MAXTRIES is unused. The other is still needed for args in CommandLineRPC

Reply:
Maybe I messed it up and forgot to remove it on CommandLineRPC too?

Reply:
And DEFAULT_GENERATED_NBLOCKS is still used in help info.

Reply:
Have tested it without DEFAULT_GENERATED_NBLOCKS in CommandLIneRPC and it failed when used **generate** without any params.